packages/postcss-syntax/src/location-preset.ts

```diff
@@ -113,12 +141,46 @@ const plugin = declare<LocationPluginOptions, PluginObj<LocationPluginState>>((a
           }
 
           state.resetLocations[declaratorId] = resetStyles.node.loc;
+
+          // For reset styles we only care about the comment directives on the variable declaration
+          // The leading commment can either be attached to a variable declaration of an export declaration (i.e. export const useResetStyles = ...)
+          const parentDeclaration =
+            path.findParent(p => p.isExportNamedDeclaration()) ?? path.findParent(p => p.isVariableDeclaration());
+          if (parentDeclaration) {
+            const commentDirectives = parseCommentDirectives(parentDeclaration.node.leadingComments);
+            if (commentDirectives) {
+              state.resetCommentDirectives ??= {};
+              state.resetCommentDirectives[declaratorId] = commentDirectives;
+            }
+          }
         }
       },
     },
   };
 });
 
+function parseCommentDirectives(leadingComments: t.Comment[] | null | undefined): CommentDirective[] | null {
+  if (!leadingComments) {
+    return null;
+  }
+
+  const entries = leadingComments
+    // We don't support comment blocks
+    .filter(comment => comment.type === 'CommentLine')
+    .map(comment => {
+      const commentValue = comment.value.trim();
+      if (!commentValue.startsWith('griffel-')) {
```

Comment:
The hardcoded strings are never used more than once per module in this PR. If we do ever support more directives, consider moving to the main babel preset with a more elaborate code around adding/parsing directives